src/test/java/org/apache/commons/imaging/color/ColorConversionsTest.java

```diff
@@ -116,4 +117,36 @@ public void testXYZ() {
             Debug.debug("cieluv_xyz", cieluv_xyz);
         }
     }
+
+    @Test
+    public void testRGBtoDin99b() {
+        for (final int rgb : SAMPLE_RGBS) {
+
+        	final ColorXyz xyz = ColorConversions.convertRGBtoXYZ(rgb);
```

Comment:
@Brixomatic it looks like there are tabs in this file. When you update this PR, could you check if there are any tabs and, please, replace by spaces?

Reply:
will do